src/server/db/rank.ts

```diff
@@ -9,7 +9,7 @@ TRUNCATE TABLE ${ranks};
 INSERT INTO ${ranks}
 SELECT
     ${users.id} AS user_id,
-    RANK () OVER (ORDER BY ${users.score} DESC) AS position
+    RANK () OVER (ORDER BY ${users.score} DESC, ${users.createdAt}) AS position
```

Comment:
Would this sort the rank so that the oldest user will have priority over taking a position?
This may be controversial down the line. A more fair approach would be to make it so the first user to reach that Rank would remain there if another player reaches the same amount of points.

Reply:
Adding a todo for this